homeassistant/components/panasonic_viera/__init__.py

```diff
@@ -150,15 +146,14 @@ def __init__(
         self._app_id = app_id
         self._encryption_key = encryption_key
 
-        self.state = None
-        self.available = False
-        self.volume = 0
-        self.muted = False
-        self.playing = True
-
-        self._control = None
+        self._control: RemoteControl | None = None
+        self.state: MediaPlayerState | None = None
+        self.available: bool = False
+        self.volume: int = 0
```

Comment:
```suggestion
        self.volume: float = 0
```

In `def _update(self)`:
```py
self.volume = self._control.get_volume() / 100
```